### PR TITLE
Adding new Adapter

### PR DESCRIPTION
**Pull request by thomas-33across**
Add new bidder Adapter for 33Across

### CODE REVIEW

**Comment by dbemiller** [comment]
hey @thomas-33across, this looks really good!

One question I had: I see a [33across adapter in Prebid.js](https://github.com/prebid/Prebid.js/blob/master/modules/33acrossBidAdapter.js)... but the `BIDDER_CODE` and [bidder params](https://github.com/prebid/Prebid.js/blob/master/modules/33acrossBidAdapter.md#sample-ad-unit-for-publishers) are different.

Did you expect this adapter to integrate nicely with Prebid.js?

If you use the same ID and Params, publishers can switch between calling you from the Client and Server very easily, by just listing you [in their s2sconfig](http://prebid.org/dev-docs/get-started-with-prebid-server.html#step-4-configure-s2s-bidder-adapters).

If your ID & Params are different, then for all intents and purposes these will be seen as completely separate bidders. This adapter will still be useful for AMP and Prebid Mobile publishers, but it won't be easy to use through Prebid.js.

It's your choice either way... I just wanted to make you aware of the tradeoffs. Let me know if you want me to review it as written.

**Comment by thomas-33across** [comment]
@dbemiller Thank you for pointing that out, I had change the bidder name to `33across` and update the code base.

File: adapters/33across/33across.go [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+	impExtJSON, err := json.Marshal(impExt)
+	if err != nil {
+		return &errortypes.BadInput{
+			Message: err.Error(),
+		}
+	}
+
+	imp.Ext = impExtJSON
+	request.Site.ID = ttxExt.SiteId
```
[the remark this answers is not in the document]
Comment (thomas-33across) [reply adapters/33across/33across.go]: the request object that pass into the `preprocess` method is already a copy of request

[the remark this answers is not in the document]
Comment (dbemiller) [reply adapters/33across/33across.go]: But `request.Site` is defined to be a pointer: https://github.com/mxmCherry/openrtb/blob/master/bid_request.go#L40

Copying the request makes a copy of the _pointer_. Two pointers with the same address still point to the same spot in memory.

**Comment by thomas-33across** [comment]
Hello, when will this MR be able to merge into master?

**Comment by hhhjort** [comment]
I expect by early next week, with a reasonable chance this week, well merged and a new release including it tagged.

**Comment by hhhjort** [comment]
Actually given that we are coming up on thanksgiving, probably best not to put out a new release tag including your adapter just before the holliday. So I would expect merged into Master by the end of the week, and in the first release post thanksgiving.

**Pull request merged**